omop_sssom.tsv

```diff
@@ -0,0 +1,49 @@
+# curie_map:
+#   omop: https://data.ohdsi.org/concept/
+#   omop_maps_to_value: https://data.ohdsi.org/maps_to_value/
+#   omoprel: https://data.ohdsi.org/relationship/
+# extension_definitions:
+# - property: omoprel:source_code_description
+#   slot_name: source_code_description
+# - property: omoprel:source_vocabulary_id
+#   slot_name: source_vocabulary_id
+# - property: omoprel:hierarchy
+#   slot_name: hierarchy
+# - property: omoprel:source_concept_class_id
+#   slot_name: source_concept_class_id
+# - property: omoprel:source_vocabulary_id
+#   slot_name: source_vocabulary_id
+# - property: omoprel:source_domain_id
+#   slot_name: source_domain_id
+# - property: omoprel:relationship_group
+#   slot_name: relationship_group
+# - property: omoprel:target_concept_code
+#   slot_name: target_concept_code
+# - property: omoprel:target_concept_value
+#   slot_name: target_concept_value
+# - property: omoprel:target_concept_name
+#   slot_name: target_concept_name
+# - property: omoprel:target_concept_id
+#   slot_name: target_concept_id
+# - property: omoprel:target_concept_class_id
+#   slot_name: target_concept_class_id
+# - property: omoprel:target_domain_id
+#   slot_name: target_domain_id
+# - property: omoprel:target_vocabulary_id
+#   slot_name: target_vocabulary_id
+# - property: omoprel:target_value_concept_code
+#   slot_name: target_value_concept_code
+# - property: omoprel:target_value_concept_value
+#   slot_name: target_value_concept_value
+# - property: omoprel:target_value_concept_name
+#   slot_name: target_value_concept_name
+# - property: omoprel:target_value_concept_class_id
+#   slot_name: target_value_concept_class_id
+# - property: omoprel:target_value_domain_id
+#   slot_name: target_value_domain_id
+# - property: omoprel:target_value_vocabulary_id
+#   slot_name: target_value_vocabulary_id
+# mapping_set_id: omop:440b9d7a-1d12-11ef-82ee-468b566a76b5
+subject_id	predicate_id	object_id	mapping_justification	mapping_provider	author_id	reviewer_id	source_code_description	source_vocabulary_id	hierarchy	source_concept_class_id	source_domain_id	relationship_group	target_concept_code	target_concept_value	target_concept_name	target_concept_id	target_concept_class_id	target_domain_id	target_vocabulary_id	target_value_concept_code	target_value_concept_value	target_value_concept_name	target_value_concept_class_id	target_value_domain_id	target_value_vocabulary_id
```

Comment:
## General observations
Hmm, interesting. I don't have the background on these OMOP post coordinated expressions, so looks foreign to me. I wonder how much of this is generalizable, so that it can be used with the ICD11 case, etc? Is there a risk that if we change things to be generalizable, OHDSI would not be happy?

There are a lot of new fields here, and I see that they are shorthand for some CURIEs in the OMOP data model. I'm not sure if this `property`/`slot_name` thing is also new with this PR; I don't see it in the docs. But I'm guessing this is not 100% tied to post-coordinated expressions, but some way of extending SSSOM.

Reply:
Its is totally generalisable! Did you read the [spec](https://github.com/monarch-initiative/uri-expression-language)? It works for everything, every system and every language.

So from OHDSI perspective, they dont care. They will say:

```
omop_maps_to_value:(target_concept_id:'4016949',target_value_concept_id:'4146067')
```

Which is there thing, their space (their post-coordination), and no one can mess with it. The same in Mondo might look like this:

```
obo_modified_disease:(disease:'MONDO:4016949',modifier:'MONDO:4146067')
```

Reply:
Oh ok, that's good to know. No I've never heard of this spec before.